### DISCUSSION TITLE
Monocle opacity

### DISCUSSION THREAD

**Discussion by barkyq** [leftwm/leftwm#1205] [Q&A]
Hi,

I am coming from DWM, where, in monocle mode, if a window had some transparency I could see the other windows open beneath it. I quite liked this, and I was wondering if it was possible to have this in leftwm.

**Comment by HighPriest** [comment]
Hello.
I am also experimenting with creating an `alacritty` scratchpad window, with reduced opacity.
But despite alacritty reducing its opacity (I can see the background color fade out with lower values), the windows underneath (background) does not render, leaving `alacritty` window black.

Maybe it\s not an issue with LeftWM, but something related to [this topic](https://discourse.nixos.org/t/picom-true-transparency-not-working/31348) for example

**Reply by Eskaan** [reply]
Do you run a compositor (i.e. picom) in the background that has transparency enabled?

**Comment by AethanFoot** [comment]
This is because in the monocle layout, all windows other than the visible one, are invisible. As I am a bit out of touch, I am not sure if you can create a custom layout where these windows aren't hidden.
